src/action.js

```diff
@@ -6,59 +6,62 @@ const registerDependencyAndroid = require('./android/registerNativeModule');
 const registerDependencyIOS = require('./ios/registerNativeModule');
 const copyAssetsAndroid = require('./android/copyAssets');
 const copyAssetsIOS = require('./ios/copyAssets');
+const pjson = require(path.join(process.cwd(), './package.json'));
```

Comment:
I don't think that's needed. I actually do not see the point of having it since it just proxies to `require`. And actually inside `getConfig` there's even no need to check for `fs.existsSync` since require will throw anyway, so you can wrap that instead.